packages/devtools_app/lib/src/scaffold.dart

```diff
@@ -470,6 +473,22 @@ class DevToolsScaffoldState extends State<DevToolsScaffold>
     );
   }
 
+  Widget _maybeWrapInShortcuts(
```

Comment:
instead of making this a method, create a widget class KeyboardShortcuts that takes required named params. And you can change `widget` to `child` to match flutter naming conventions. The KeyboardShortcuts widget can handle the logic here in its build method

Reply:
Makes sense, switched it to a widget